Jenkinsfile

```diff
@@ -41,13 +41,7 @@ pipeline {
 
     triggers {
         /* groovylint-disable-next-line AddEmptyString */
-        cron(env.BRANCH_NAME == 'master' ? 'TZ=America/Toronto\n0 0 * * *\n' : '' +
-             /* groovylint-disable-next-line AddEmptyString */
-             env.BRANCH_NAME == 'release/1.2' ? 'TZ=America/Toronto\n0 12 * * *\n' : '' +
-             /* groovylint-disable-next-line AddEmptyString */
-             env.BRANCH_NAME.startsWith('weekly-testing') ? 'H 0 * * 6\n' : '' +
-             /* groovylint-disable-next-line AddEmptyString */
-             env.BRANCH_NAME.startsWith('md-on-ssd-testing') ? 'H 12 * * 0' : '')
+        cron(env.BRANCH_NAME.startsWith('md-on-ssd-testing') ? 'H 12 * * *' : '')
```

Comment:
Is `startswith()` the test we want here?

Reply:
Given this file only runs on this branch I don't think it should matter, but I've updated it.

Reply:
I'm not convinced that `cron()` entries are isolated to branches but this change also prevents somebody from creating `md-on-ssd-testing-my-test` (although I do think we have guards against that, but still... better to be proactive) as a fork of this branch and inheriting the `cron()`